week-one/Course.java

```diff
@@ -0,0 +1,91 @@
+import java.util.ArrayList;
+
+public class Course {
+  String myName;
+  ArrayList<Nano> myStudents = new ArrayList<Nano>();
+  private ArrayList<Mentor> myMentors = new ArrayList<Mentor>();
+
+  public Course(String name) {
+    myName = name;
+  }
+  public void addStudent(Nano student) {
+    myStudents.add(student);
+  }
+
+  public void setMentor(Mentor mentor) {
+   if (myMentors.size() != 0) {
+     while (myMentors.size() != 0) {
+       myMentors.remove(0);
+     }
+   }
+   myMentors.add(0, mentor);
+  }
+
+  public void setMentors(Mentor[] mentors) {
+    if (myMentors.size() != 0) {
+     while (myMentors.size() != 0) {
+       myMentors.remove(0);
+     }
+   }
+    for (int i = 0; i < mentors.length; i++) {
+      myMentors.add(mentors[i]);
+    }
+  }
+
+  public int getClassSize() {
+    return myStudents.size();
+  }
+
+  public int getMentorSize() {
+    return myMentors.size();
+  }
+
+  public void removeMentor(Mentor mentor) {
+    for (int i = 0; i < myMentors.size(); i++) {
+      if (myMentors.get(i) == mentor) {
+        myMentors.remove(i);
+      }
+    }
+  }
+
+  public void removeStudent(Nano student) {
+    for (int i = 0; i < myStudents.size(); i++) {
+      if (myStudents.get(i) == student) {
+        myStudents.remove(i);
+      }
+    }
+  }
+
+  public boolean isNanoInTheClass(Nano student) {
```

Comment:
Usually the variable name should be the same name as the class name, so in this case `Nano nano`.